src/main/resources/file.csv

```diff
@@ -0,0 +1,8 @@
+b,banana,20
```

Comment:
I belive both - this file and result file should have headers on top (e.g "type,fruit,quantity" and "fruit,quantity"). It was written like that in readme file